### PR TITLE
lint: clean warn

### PR DESCRIPTION
**Pull request by zirain**
Signed-off-by: hejianpeng <hejianpeng2@huawei.com>

### CODE REVIEW

**Comment by Xunzhuo** [comment]
Still have: 
image

**Comment by zirain** [comment]
fixed
```
===========> Running lint.yamllint ... 
tools/bin/yamllint --config-file=tools/linter/yamllint/.yamllint $(git ls-files :*.yml :*.yaml | xargs -L1 dirname | sort -u) 
===========> Running lint.codespell ... 
```

File: internal/xds/translator/testdata/in/xds-ir/multiple-listeners-same-port.yaml [hunk 1]
```diff
@@ -1,66 +1,70 @@
...
-  address: "0.0.0.0"
-  port: 10080
-  hostnames:
-  - "example.net"
-  routes:
-  - name: "fourth-route" 
-    destinations:
-    - host: "1.2.3.4"
-      port: 50000
+  - name: "first-listener"
```
Comment (arkodg) [inline internal/xds/translator/testdata/in/xds-ir/multiple-listeners-same-port.yaml:2]: can you revert the identation or was it generated ?

Comment (zirain) [reply internal/xds/translator/testdata/in/xds-ir/multiple-listeners-same-port.yaml:2]: looks like I run `Format Document` with vscode, let me revert this.

**Review by arkodg — APPROVED** [review]
LGTM thanks !

**Pull request merged**